core/src/trezor/wire/__init__.py

```diff
@@ -150,12 +128,16 @@ async def wait(self, *tasks: Awaitable) -> Any:
 
 PROTOBUF_BUFFER_SIZE = 8192
 
+WIRE_BUFFER = bytearray(PROTOBUF_BUFFER_SIZE)
+if __debug__:
+    WIRE_BUFFER_DEBUG = bytearray(PROTOBUF_BUFFER_SIZE)
+
 
 class Context:
-    def __init__(self, iface: WireInterface, sid: int) -> None:
+    def __init__(self, iface: WireInterface, sid: int, buffer: bytearray) -> None:
         self.iface = iface
         self.sid = sid
-        self.buffer = bytearray(PROTOBUF_BUFFER_SIZE)
+        self.buffer = buffer
```

Comment:
Would it make sense to zero the buffer here, just to be on the safe side?

Reply:
there is no reason, the buffer is write-only from Context's point of view.

ceb21517 clarifies this -- aparently there's some leftovers that were never even used